sdk/formrecognizer/ai-form-recognizer/src/bin/gen-model.ts

```diff
@@ -1,25 +1,23 @@
-#!/bin/env node
+#!/bin/env node;
```

Comment:
semicolon should be removed?

also would be helpful to exclude this from the linting rules similar to https://github.com/Azure/azure-sdk-for-js/blob/a2ca1203f900d146d4054508c1aebfebb762e24f/sdk/apimanagement/api-management-custom-widgets-scaffolder/eslint.config.mjs#L11-L17

Reply:
thanks Jeremy